### ISSUE TITLE
Make Link Hints work together with Vim plugins

### ISSUE THREAD

**Issue by lydell** [lydell/LinkHints#6]
Some Vim plugins:

- [Vimium](https://github.com/philc/vimium) and [Vimium C](https://github.com/gdh1995/vimium-c)
- [Tridactyl](https://github.com/tridactyl/tridactyl)
- [Vim Vixen](https://github.com/ueokande/vim-vixen)

It would be cool if it was possible to:

1. Trigger Link Hints commands from Vim shortcuts in one of those plugins.
2. Suspend keyboard processing in the Vim plugin while using hints in Link Hints (so you don’t activate Vim commands while trying to use hints).
3. Re-enable the Vim plugin when done hinting in Link Hints.

This requires collaboration with the Vim plugins, though. Maybe there’s some other integration possibility as well that I haven’t thought of.

**Comment by alexherbo2** [comment]
Krabby uses a lot the [cross-extension messaging] for its functionalities, and also [Saka Key] to tab search with [Saka].

[Cross-extension messaging]: https://developer.chrome.com/extensions/messaging#external
[Saka Key]: https://key.saka.io
[Saka]: https://saka.io

**Comment by lydell** [comment]
People who have voted :+1: – which Vim extensions are you most interested in?

**Comment by imsodin** [comment]
Currently i am using vimium with enough keys ignored to be able to use link hints for links.

**Comment by lindhe** [comment]
Only Vimium.

**Comment by bovine3dom** [comment]
I don't think you need to do 2 & 3: if you just provide a way to trigger hinting and tell us when you are done (ideally optionally doing nothing to the page and just returning information about what was selected) we can do the rest - or perhaps you or intrepid users can via PRs to the extensions.

(I suspect there may be no-one who uses both LinkHints and Tridactyl but I'm quite keen for more interop between extensions so it could be an useful exercise).

**Comment by braham-snyder** [comment]
> People who have voted 👍 – which Vim extensions are you most interested in?

Tridactyl

**Comment by mawkler** [comment]
I'm currently just using Vimium. Could someone please explain how LinkHints differs from Vimium's `f` functionality?
